AppController/djinn.rb

```diff
@@ -3165,8 +3160,7 @@ def backup_appcontroller_state()
       Djinn.log_debug("backup_appcontroller_state: no changes.")
       return
     end
-
-    Djinn.log_debug("backup_appcontroller_state:"+local_state.to_s)
+    Djinn.log_debug("backup_appcontroller_state saved changed state.")
```

Comment:
Would it be better to print the backed up state but with truncating the sensitive info? Would it help with debugging you think?